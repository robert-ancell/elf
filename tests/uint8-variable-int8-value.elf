int8 v = 127
uint8 value = v
print (value)
