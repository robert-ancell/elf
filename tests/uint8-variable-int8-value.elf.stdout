Line 2:
uint8 value = v
      ^^^^^
Variable is of type uint8, but value is of type int8
